### PR TITLE
Add Categories Section

### PR DESCRIPTION
**Pull request by Bondok6**
# Hello there 🙋‍♂️,

Thanks for taking the time to review our project.

<hr>

## Changes 🧾

- Add Categories section (*User can choose any category he wants*).
- Add `renderCategories.js` file that contains `renderCategories()` function.
- Update `getMeals.test.js` file.

### CODE REVIEW

**Review by andres-condezo — APPROVED** [review]
# Hi @Bondok6  :wave: 

## Status: Approved ✔️ 

Great job so far! Thank you for contributing to improving the application.

## Highlights
- Congratulations on the good job implementing the categories section  ✔️.
- No linter errors ✔️
- Descriptive pull request ✔️ 

 

## Suggestions
- [optional] I would recommend you not to forget to create a new branch to implement a new feature so that once the changes of your new branch are approved, it can be merged with the developer branch and after a second review, it can be merged with the main branch.

---

There is nothing else to say other than... it's time to merge it.
Congratulations! 🎉 

### Cheers and Happy coding!👏👏👏

**Comment by Bondok6** [comment]
### Thanks @andres-condezo for your recommendation, you're right I'll do it.

**Pull request merged**
